tests/keras/test_ops.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import absolute_import
+from __future__ import print_function
+import pytest
+import os
+import numpy as np
+np.random.seed(1337)
+
+from keras import backend as K
+from keras.models import Model
+from keras.layers import Dense, Input, merge, Lambda
+
+
+def test_ops_1():
```

Comment:
Please fix the identations issues in the file. I'll review the PR in the next few days. Thanks for updating it!